package.json

```diff
@@ -46,12 +46,15 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.0.0",
+    "@microsoft/eslint-plugin-sdl": "^0.1.8",
     "@rnx-kit/eslint-plugin": "^0.2.0",
     "@rnx-kit/jest-preset": "^0.1.0",
     "@types/jest": "^27.0.0",
     "@types/node": "^16.0.0",
     "eslint": "^8.0.0",
     "eslint-plugin-jest": "^26.0.0",
+    "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-security": "1.4.0",
```

Comment:
Waiting for https://github.com/microsoft/eslint-plugin-sdl/pull/24 to be published so we can remove these two lines.